src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,14 +1,28 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { actions as actionsFilter } from '../../features/filter';
+import { useAppSelector } from '../../app/hooks';
 
 export const TodoFilter: React.FC = () => {
+  const { status, query } = useAppSelector(state => state.filter);
+  const dispatch = useDispatch();
+
+  const deleteHandler = () => {
```

Comment:
```suggestion
  const handleDelete = () => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364

Reply:
Еhanks for the link this is such an interesting article